dvc/info.py

```diff
@@ -134,3 +142,26 @@ def _get_dvc_repo_info(self):
         return "dvc (subdir), git"
 
     return "dvc, git"
+
+
+def _get_external_remotes_cache(self):
+    cache_info = self.config.get("cache", {})
+
+    cache_vars = [
+        val for val in Schemes.__dict__.values() if isinstance(val, str)
+    ]
+    cache_key = list(set(cache_info.keys()).intersection(set(cache_vars)))
+
+    core_info = self.config.get("core", {})
+    cache_location, remote_url = "", ""
+
+    if cache_info and cache_key:
+        cache_location = cache_info[cache_key[0]]
+
+    if core_info.get("remote", False):
+        remote = core_info["remote"]
+        remote_url = (
+            self.config.get("remote", {}).get(remote, {}).get("url", None)
+        )
```

Comment:
Seems like `cache` and `remote` parts are not really related in this helper, so could be separated into separate helpers.

Reply:
I agree with you here, I will make changes and push.